### ISSUE TITLE
New support

### ISSUE THREAD

**Issue by scindo07** [Zlkcyber/teneo-farm#3]
Can't getting the account using access_tokens instead? or id?

**Comment by tructhienbui** [comment]
yeah update require to include accessToken along, but those was very simple fix
